vspace/src/main/webapp/WEB-INF/views/staff/modules/module.jsp

```diff
@@ -1,9 +1,88 @@
+<%@ page pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
 
-	<h1>Module: ${module.name}</h1>
-	<h3>Description: ${module.description}</h3>
+
+<script>
+$( document ).ready(function() { 
+	
+	<c:forEach items="${slides}" var="link" varStatus="loop">
+	{		
+        $('table tr:last')
+        
+            .after('<tr><td style="padding-left: 22px; border-style:hidden; padding-top: 6px; padding-bottom: 6px;"><a href="<c:url value="/staff/module/slide/${link.id}" />">'+     
+            '<div class="card" style="max-width: 18rem;">' +          
+            '<div align="left" class="card-body"><h5 class="card-title">${link.name}</h5>'+                     
+              '<p class="card-text">${link.description}</p>'+
+            '</div>'+
+          '</div>'+ 
+          '</td>');
```

Comment:
why is that not a normal foreach loop that just prints out table rows? Why is this done with jquery?

Reply:
?